numpy/lib/_function_base_impl.pyi

```diff
@@ -505,6 +505,7 @@ def percentile(
     overwrite_input: bool = ...,
     method: _MethodKind = ...,
     keepdims: L[False] = ...,
+    weights: None | _ArrayLikeFloat_co = ...,
```

Comment:
```suggestion
    *,
    weights: None | _ArrayLikeFloat_co = ...,
```
A minor comment: all `weights` parameters should be annotated as keyword-only. The rest looks perfectly fine though.